src/components/archive.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Link } from './';
+import Icon from './icon';
+
+export default function Archive({ title, href, ...props }) {
+  return (
+    <Link to={href} title={title} className="small" {...props}>
+      <PdfIcon /> {title}
+    </Link>
+  );
+}
+
+const PdfIcon = ({ className }) => (
```

Comment:
What's the `className` prop about?

Reply:
I think it can be removed since no extra styling is needed for this case

Reply:
removed